pages/twitter.tsx

```diff
@@ -152,23 +152,20 @@ const Twitter: NextPage = () => {
   const errorScreen = isConnected && screen === "error";
 
   return (
-    <div className={styles.screen}>
-      <div className={styles.wrapperScreen}>
-        <div className={styles.container}>
+    <div className={homeStyles.screen}>
+      <div className={homeStyles.wrapperScreen}>
+        <div className={homeStyles.container}>
           {screen === "verifyTwitter" &&
             (!isConnected ? (
               <h1 className="sm:text-5xl text-5xl">You need to connect anon</h1>
             ) : (
-              <>
-                <h1 className="sm:text-5xl text-5xl mt-4">
-                  It&apos;s time to verify your twitter on chain !
-                </h1>
-                <div className="mt-8">
-                  <Button disabled={Boolean(!calls)} onClick={verifyTwitter}>
-                    Verify my Twitter
-                  </Button>
-                </div>
-              </>
+              <VerifyFirstStep
+                onClick={verifyTwitter}
+                disabled={Boolean(!calls)}
+                buttonLabel="Verify my Twitter"
+                title="It's time to verify your twitter on chain !"
+                subtitle="Safeguard your account with our network verification page"
+              />
```

Comment:
You need to make it on all social media verifications

Reply:
I did it for Twitter, Github and Discord. What are the other social medias ?

Reply:
It's all of them :)